core/sql/optimizer/BindRelExpr.cpp

```diff
@@ -11046,6 +11046,14 @@ RelExpr *MergeDelete::bindNode(BindWA *bindWA)
         bindWA->setErrStatus();
         return this;
       }
+      if (CmpCommon::getDefault(COMP_BOOL_175) == DF_OFF)
+      {
+        // MERGE DELETE + INSERT is buggy, so disallow it unless CQD is on. In
```

Comment:
Since COMP_BOOL_175 is OFF by default, this check prevents MERGE_DELETE + INSERT.  Is it intended?